### DISCUSSION TITLE
Is is possible to disable the fan when the internal unit is idle (heat mode)?

### DISCUSSION THREAD

**Discussion by enomam** [echavet/MitsubishiCN105ESPHome#94] [General]
Hey all,

My MSZ-AP units, once they reach the desired temperature, they switch to idle mode. In idle mode, though the compressor stops running, the fan continues to run. I understand that this is to maintain airflow, so the internal thermostat has an up-to-date temperature. The side effect of this though, is that it makes the room drafty.

Is there a way to disable this function, especially as some of us use external thermostats? I've seen (theoretically) that it can be disabled by snipping the JR01 jumper - but was wondering if there's a less drastic solution via software?

**Comment by echavet** [comment]
Hi,
Sure it sounds possible to me. I'm not really sure it is a good idea to
interfere with the heatpump thermostat (as I would not exclude that the air
flow is thermally neutral).
But you could try.
I think it could be handled by a HA automation.
You would track for state changes on the stage sensor
(ex:sensor.esp32_hp_stage_sensor is you named it "stage sensor"), and call
Climate services to force the fan speed.
As far as I remember, it seems to me that my heatpump slows the fan very
low when it is in idle mode. But the silent mode could be again slower.

**Comment by enomam** [comment]
I think the control I'm looking for is called " Thermal Off Fan Operation (Heat Mode)".
I think it's low by default (which causes drafts), so want to set it to off (there's no "off" fan speed :( )

Also - the "stage sensor" is low when the unit idles (and, is sometimes low when heating too :/ )

![image](https://github.com/echavet/MitsubishiCN105ESPHome/assets/7985914/33fcfc9e-0ff8-4661-b494-090d5bd3da44)

![image](https://github.com/echavet/MitsubishiCN105ESPHome/assets/7985914/8c9e25cb-2831-48ab-8b41-201d264e9b67)

**Comment by enomam** [comment]
You know what - I think it's related to the stuff being investigated here: https://github.com/echavet/MitsubishiCN105ESPHome/wiki/Dictionary-of-Messages-on-CN105-Serial

and here:
https://github.com/SwiCago/HeatPump/issues/39

**Reply by echavet** [reply]
Oh yes, that's the "paleontologist" branch of the project! 😜
It's pure research!! I'm very proud of the contributions from @disruptivepatternmaterial to the project. There are plenty of things to discover about this protocol. And plenty of things that I don't master at all!
Perhaps @disruptivepatternmaterial will have a better Idea about your question.
Look at the documentation he published here:
"Setting 125: When in Heat mode, once set point has been reached, fan 1) stays on, or 2) turns off
Value: 1 (default) – fan stays ON once set point is reached
2 – fan turns OFF once set point is reached" 
https://github.com/echavet/MitsubishiCN105ESPHome/discussions/39#discussioncomment-8558411

**Reply by disruptivepatternmaterial** [reply]
Yes some units have settings of what to do in these states. Mine can do that. For the record, it turns off the head unit fan, not the outside one - that is independent.

I think if you follow some of the notes between @echavet and I on those settings, I am concerned that the dictionary is really different from unit to unit. Keep in mind I have in this house and they are all of the same general type and I live in North America.

I kind of stopped working on that part too because I really was after trying to find how to change the +/-2C of auto, which I was unable to find...instead I started working on an HA auto...

**Reply by enomam** [reply]
Sorry for the delay.
Is there any testing/debug I can do on my end to establish the "dictionary" for my unit, and where (if?) "setting 125" can be controlled via software?

**Reply by Tommzy** [reply]
Hi! have you find any clue to toggle the setting in the code?

**Comment by Tommzy** [comment]
Setting 127 is a great feature that can block `backwards humidify`. It's a pretty useful setting under the weather before/in the middle of the early rain season. (high humidity + relative low temp but feeling temp is high). The unit(new machine does too)  would cause high humidity smell in the room
Here's a discussion: 
https://diy.stackexchange.com/questions/143875/how-to-i-eliminate-the-smell-coming-from-my-mitsubishi-mini-split-units 

I assume Setting 125 will be helpful to prevent the room super dry maybe?
